### ISSUE TITLE
Crash report

### ISSUE THREAD

**Issue by zzlazz** [Tobi823/ffupdater#110]
Stacktrace:
```
java.lang.IllegalStateException: PackageInfo for file '/storage/emulated/0/Android/data/de.marmaro.krt.ffupdater/files/Download/com.brave.browser.apk' is null.
	at de.marmaro.krt.ffupdater.download.PackageManagerUtil$getPackageArchiveInfo$2.invokeSuspend(PackageManagerUtil.kt:26)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
```
Device information:
| key | value |
| --- | --- |
| version name | 75.0.0 |
| version code | 91 |
| build type | release |
| SDK version | 29 |
| android release | 10 |
| board | exynos9810 |
| brand | samsung |
| device | starlte |
| manufacturer | samsung |
| model | SM-G960F |
| product | starltexx |
| SOC manufacturer | / |
| SOC model | / |
| supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| tags | release-keys |
| build time | 1634610178000 |

**Comment by Tobi823** [comment]
I will take a closer look at the problem after the university exams.

**Comment by zzlazz** [comment]
Good luck!

**Comment by Tobi823** [comment]
Should be fixed in version 75.1.0

**Comment by zzlazz** [comment]
thanks!
